README.md

```diff
@@ -37,9 +37,6 @@ This sample allows bidders to confidentially submit their bids to an enclave (A
 ### [Conclave Corda Trade](./conclave-corda-trade):
 This application serves as a demo for building a confidential trading system based on Corda and Conclave. An exchange Corda node would serve as a host which runs the enclave, while broker nodes serve as clients which send encrypted orders from their end-clients which are matched in the enclave and trades generated are recorded in all relevant participants ledgers.
 
-### [Conclave Hello World](./hello-world):
-This is a simple conclave application which shows you how you can reverse a string in java.
-
 ### [Conclave Hello World Python](./hello-world-python):
```

Comment:
Should we not remove this as well?

Reply:
That was added by @snedamle recently. It's not referenced by the Docsite and separate to the hello world.

Reply:
I believe Sneha was adding hello world sample with python to show how python can be used in Conclave. It's DevRel's sample.